services/docker-compose.yml

```diff
@@ -198,6 +198,7 @@ services:
       - SIMCORE_SERVICES_NETWORK_NAME=interactive_services_subnet
       - TRACING_THRIFT_COMPACT_ENDPOINT=${TRACING_THRIFT_COMPACT_ENDPOINT}
       - DYNAMIC_SIDECAR_PROMETHEUS_SERVICE_LABELS=${DYNAMIC_SIDECAR_PROMETHEUS_SERVICE_LABELS}
+      - DYNAMIC_SIDECAR_PROMETHEUS_MONITORING_NETWORKS=${DYNAMIC_SIDECAR_PROMETHEUS_MONITORING_NETWORKS:-[]}
```

Comment:
Let's use defaults only in Python code.

Reply:
I don't understand why. Could you elaborate?

Reply:
To avoid having too many places where the value may come from. Right now we already have quite a few:
* Dockerfile
* Python code 
* repo.config

Before (and it is still the case to some extent) we had a "mess" with ENV variables since they were coming from everywhere. Please, read exact [description here](https://github.com/ITISFoundation/osparc-simcore/issues/3635#issuecomment-1628329002).

So, right now we are trying to keep our ENV clean